api/routes/init.go

```diff
@@ -6,6 +6,6 @@ import (
 )
 
 func SetupRoutes(engine *gin.Engine) {
-	engine.GET("/zones", controllers.HandleZonesGet)
 	engine.POST("/signup", controllers.HandleSignUp)
+	engine.POST("/nearzones", controllers.HandleNearGyms)
```

Comment:
I think an endpoint like /near_gyms would be clearer.

Reply:
@PedroChaparro Solved!